bats/tests/containers/published-udp-ports.bats

```diff
@@ -0,0 +1,42 @@
+load '../helpers/load'
+
+@test 'factory reset' {
+    factory_reset
+}
+
+@test 'start container engine' {
+    start_container_engine
+    wait_for_container_engine
+}
+
+run_container_with_published_udp_port_and_connect() {
+    local ip=$1
+    local port=$2
+    local netcat_connect_addr=$3
+    ctrctl run -d --name socat-udp-"$port" -p "$ip":"$port":"$port"/udp alpine sh -c \
+        "apk add --no-cache socat && socat -v -T1 UDP-RECVFROM:$port,fork STDOUT"
+    sleep 5
+    echo "hello from nc UDP port $port" | nc -u -w1 "$netcat_connect_addr" "$port"
+    run ctrctl logs socat-udp-"$port"
+    assert_success
+}
```

Comment:
Shouldn't we also stop (and remove) the container? Or are we assuming the user will clean it up later?

Reply:
We generally assume that the user will perform a factory-reset after running BATS.